src/config.ts

```diff
@@ -69,6 +72,11 @@ const validators = {
   REACT_APP_PENDO_VISITOR_ID_PREFIX: envalid.str({ default: 'test_visitor' }),
 }
 
+const constants = {
```

Comment:
is this to avoid a circular dependency, or can you explain the rationale here? this config file up until this point is to manage/sanitize/validate `.env` vars - mixing in constants here feels a little weird.

Reply:
Yes. It was previously in AppProviders and several files in plugins imported it creating a circular dependency. Another option would be make a separate file with just these types of constants. Given there is only one that I found, I chose the simple path. I'm not married to it.

Reply:
I added a TODO comment here for future reference.